ui_framework/components/form/index.js

```diff
@@ -0,0 +1 @@
+export { KuiColorPicker } from './color_picker';
```

Comment:
Can we move this component into its own `color_picker` directory? It's such a rich component, I feel like it might make more sense to make it more top-level and leave the `form` directory for more primitive form components.